File: app/src/main/java/com/abqwtb/model/BusTrip.java
package com.abqwtb.model;

import org.joda.time.LocalTime;

public class BusTrip {

  public LocalTime scheduledTime;
  public int route;
  public float secondsLate;
  public int busId;

  @Override
  public String toString() {
    return scheduledTime.toString("h:mm aa");
  }
}

File: app/src/main/java/com/abqwtb/model/BusTrip.kt
package com.abqwtb.model

import org.joda.time.LocalTime

class BusTrip {

    var scheduledTime: LocalTime? = null
    var route: Int = 0
    var secondsLate: Float = 0.toFloat()
    var busId: Int = 0

    override fun toString(): String {
        return scheduledTime!!.toString("h:mm aa")
    }
}
